lib/Distribution/Sle/16Latest.pm

```diff
@@ -0,0 +1,53 @@
+# SUSE's openQA tests
+#
+# Copyright 2019-2022 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: The class represents current (i.e. latest) Sle15 distribution and
+# provides access to its features.
+# Follows the "Factory first" rule. So that the feature first appears in
+# Tumbleweed distribution, and only if it behaves different in Sle15 then it
+# should be overriden here.
```

Comment:
This is copy/paste, doesn't apply.

Reply:
copied from agama devel, not sure fit or not.